tests/dune

```diff
@@ -11,3 +11,14 @@
 (alias
   (name runtest)
   (deps merlin-wrapper))
+
+(include dune.inc)
+
+(rule
+ (targets dune.inc.gen)
+ (deps    (source_tree .))
+ (action  (with-stdout-to %{targets} (run ./gen/gen_tests.exe))))
+
+(alias
+ (name   runtest)
+ (action (diff dune.inc dune.inc.gen)))
```

Comment:
Ddi you try `(mode promote)` in the `rule` above and change the target to `dune.inc`? That seems a bit simpler.

Reply:
I just copied what dune itself is doing.
I think there was a reason why simply generating `dune.inc` wouldn't work, but I don't remember. Perhaps @aalekseyev will know?